style.css

```diff
@@ -97,34 +109,107 @@ input#search-input {
   box-shadow: 0 1px 5px 2px rgba(0, 0, 0, 0.3);
 }
 
-input#search-input:hover {
-  width: 350px;
-}
-
 #card-body {
   display: flex;
   flex-wrap: wrap;
   justify-content: center;
-  width: 1550px;
+  width: 100%;
   margin: 0 auto;
-  padding: 2rem;
+  margin: 2rem;
```

Comment:
By putting the width to 100% and the margin to 2rem, it will mean that the card-body div will be larger than the screen size, so the bottom scroll bar will appear on the page/screen.I'll send a screenshot of what I mean on slack :)